go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	github.com/streadway/quantile v0.0.0-20150917103942-b0c588724d25 // indirect
 	github.com/tsenart/go-tsz v0.0.0-20180814235614-0bd30b3df1c3 // indirect
 	github.com/tsenart/vegeta v12.7.0+incompatible
+	github.com/vmware-tanzu/velero v1.4.0
```

Comment:
We'll need to do a go mod vendor update as well while we're here and check in vendored code. I know this isn't how your team probably does this, but for now this is our process.